### DISCUSSION TITLE
Mod functionality

### DISCUSSION THREAD

**Discussion by AaronYu0625** [Lukium/palworld-servertools#19] [Ideas]
I wish to add the buttons to open the Mods and LogicMods folders, making it more convenient for users to manage mods.

**Comment by Lukium** [comment]
> I wish to add the buttons to open the Mods and LogicMods folders, making it more convenient for users to manage mods.

The app now, on Windows, gives you the option to setup several shortcuts, including to the PAK directory, Binaries, and a few others. Let me know if this is sort of what you're looking for
